test/cases/ostree.sh

```diff
@@ -33,6 +33,12 @@ case "${ID}-${VERSION_ID}" in
         OS_VARIANT="rhel8-unknown"
         COMPOSE_URL="${COMPOSE_URL:-http://download.devel.redhat.com/rhel-8/nightly/RHEL-8/latest-RHEL-8.4}"
         BOOT_LOCATION="$COMPOSE_URL/compose/BaseOS/x86_64/os/";;
+    "rhel-8.5")
+        IMAGE_TYPE=edge-commit
+        OSTREE_REF="rhel/8/${ARCH}/edge"
+        OS_VARIANT="rhel8-unknown"
+        COMPOSE_URL="${COMPOSE_URL:-http://download.devel.redhat.com/rhel-8/nightly/RHEL-8/latest-RHEL-8.5}"
```

Comment:
This is in conflict/at odds with https://github.com/osbuild/osbuild-composer/issues/1397 and also partially with https://github.com/osbuild/osbuild-composer/pull/1274. Need to be careful not to end up with locations pointing both to 8.4 and 8.5 nightlies.

Reply:
Please rebase again and drop line 40.

Reply:
Done